### ISSUE TITLE
Php and mysql

### ISSUE THREAD

**Issue by Rubemlrm** [gsilvapt/docker-compose-materials#7]
One of the most common case samples we can create, is using docker-compose to create a scenario of an application running Php with a connection to a database to mysql/mariadb.

**Comment by gsilvapt** [comment]
Yes, we can surely create something with that stack. I haven't though throughout all stacks I want to provide with examples. However, when I start digging in with examples of how to use Docker and Docker-compose in development workflows, I sure want to use complete stacks to prove the points. At that time, we can surely do something in that stack, although I never coded in PHP (I'm more acostumed to Java, Python, NodeJS). You're free to create those anyway and push to the repository! :smile:
